### ISSUE TITLE
tokio::task::spawn_blocking() tasks show as 'warning: task has never yielded' 

### ISSUE THREAD

**Issue by capveg-netdebug** [tokio-rs/console#528]
### What crate(s) in this repo are involved in the problem?

tokio-console

### What is the issue?

I think this is just a display issue, but IMHO an (intentionally) blocking task, e.g., from `spawn_blocking` shouldn't show up as a warning.

### How can the bug be reproduced?

```rust

use std::time::Duration;

#[tokio::main]
async fn main() {
    console_subscriber::init();
    tokio::task::spawn_blocking( move || {
        loop { std::thread::sleep(Duration::from_secs(10))}
    });
}

```

See screenshot
![Screenshot from 2024-02-20 11-01-26](https://github.com/tokio-rs/console/assets/135524607/722c8cc7-1fcb-47ee-886f-41f9cfebc005)

### Versions

```text
[robs@LAPTOP-O7RS71IJ tokio-console-repro]$ cargo tree | grep console
tokio-console-repro v0.1.0 (/home/robs/git/tokio-console-repro)
├── console-subscriber v0.2.0
│   ├── console-api v0.6.0
```

### Possible solution

Test if the task is a task_blocking before warning.

### Additional context

Great project - thank you for it!

### Would you like to work on fixing this bug?

yes

**Comment by hds** [comment]
Thanks for your issue submission. It's lovely and clear.

This is a duplicate of #516, and even better it has been fixed in the PR #517.

That hasn't gone out for a release yet, but we're hoping to get a new release out before too long.

I'll close this as a duplicate. Please comment if you think we've missed something.

**Comment by capveg-netdebug** [comment]
Thank you for the fast reply!  FYI: I did see #516 but it wasn't clearly the same thing so I filed anyway (and forgot to reference #516).  Looking forward to the fix being released and thank you again for the great tool!
